experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -0,0 +1,531 @@
+    parent(p.canopy.energy.turbulent_fluxes.shf) .= parent(turb_energy_flux_canopy.shf)
+    parent(p.snow.turbulent_fluxes.lhf) .= parent(turb_energy_flux_snow.lhf)
+    parent(p.snow.turbulent_fluxes.shf) .= parent(turb_energy_flux_snow.shf)
+    parent(p.soil.turbulent_fluxes.lhf) .= parent(turb_energy_flux_soil.lhf)
+    parent(p.soil.turbulent_fluxes.shf) .= parent(turb_energy_flux_soil.shf)
+end
+
+"""
+    update_field!(sim::ClimaLandSimulation, ::Val{:turbulent_moisture_flux}, fields::NamedTuple)
+
+Takes in a NamedTuple having fields `canopy`, `snow`, and `soil`, and updates
+each component of the land simulation with the corresponding field.
+
+The soil moisture flux is split into liquid and ice components, `Ẽ_l` and `Ẽ_i`.
+"""
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:turbulent_moisture_flux}, fields::NamedTuple)
+    turb_moisture_flux_canopy = fields.canopy
+    turb_moisture_flux_snow = fields.snow
+    turb_moisture_flux_soil_liq = fields.soil.Ẽ_l
+    turb_moisture_flux_soil_ice = fields.soil.Ẽ_i
+
+    p = sim.integrator.p
+    parent(p.canopy.energy.turbulent_fluxes.transpiration) .= parent(turb_moisture_flux_canopy)
+    parent(p.snow.vapor_flux) .= parent(turb_moisture_flux_snow)
+    parent(p.soil.vapor_flux_liq) .= parent(turb_moisture_flux_soil_liq)
+    parent(p.soil.vapor_flux_ice) .= parent(turb_moisture_flux_soil_ice)
+end
+
+# Update fields stored in land drivers
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:air_temperature}, field)
+    parent(sim.integrator.p.drivers.T) .= parent(field)
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:air_pressure}, field)
+    parent(sim.integrator.p.drivers.P) .= parent(field)
+end
+function Interfacer.update_field!(::ClimaLandSimulation, ::Val{:air_humidity}, field)
+    parent(sim.integrator.p.drivers.q) .= parent(field)
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:c_co2}, field)
+    sim.integrator.p.drivers.c_co2 .= field
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:liquid_precipitation}, field)
+    ρ_liq = (LP.ρ_cloud_liq(sim.model.soil.parameters.earth_param_set))
+    parent(sim.integrator.p.drivers.P_liq) .= parent(field ./ ρ_liq)
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:snow_precipitation}, field)
+    ρ_liq = (LP.ρ_cloud_liq(sim.model.parameters.earth_param_set))
+    parent(sim.integrator.p.drivers.P_snow) .= parent(field ./ ρ_liq)
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:lw_d}, field)
+    parent(sim.integrator.p.drivers.LW_d) .= parent(field)
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:sw_d}, field)
+    parent(sim.integrator.p.drivers.SW_d) .= parent(field)
+end
+function Interfacer.update_field!(sim::ClimaLandSimulation, ::Val{:zenith_angle}, field)
+    parent(sim.integrator.p.drivers.θs) .= parent(field)
```

Comment:
I think this needs to be 
```suggestion
    parent(sim.integrator.p.drivers.cosθs) .= parent(field)
```
to match the change in ClimaLand